frontend/amundsen_application/static/js/features/HomePageWidgets/BadgesWidget/index.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+
+import BadgesList, { BadgesListContainerProps } from 'features/Badges';
+
+type BadgesWidgetProps = BadgesListContainerProps;
+
+export const BadgesWidget: React.FC<BadgesWidgetProps> = (
+  props: BadgesWidgetProps
+) => (
+  <div>
```

Comment:
Are these divs necessary?

Reply:
Nope, only the SearchBarWidget one I think